ext/standard/var.c

```diff
@@ -992,6 +1016,30 @@ static void php_var_serialize_intern(smart_str *buf, zval *struc, php_serialize_
 				zend_bool incomplete_class;
 				uint32_t count;
 
+				if (ce->ce_flags & ZEND_ACC_ENUM) {
+					PHP_CLASS_ATTRIBUTES;
+					zval obj;
+
+					zend_object *zobj = Z_OBJ_P(struc);
+					ZVAL_OBJ_COPY(&obj, zobj);
```

Comment:
What's the reason for adding `obj` - This seems like it increments the reference count(ZVAL_OBJ_COPY), then decreases the reference count(zval_ptr_dtor), but `obj` doesn't have any other uses and I'd assume the refcount would be positive even without this change

Reply:
Fixed locally